bin/create-continst.sh

```diff
@@ -99,12 +99,20 @@ CLUSTER_NAME=$1
   fi
   USERDATA=$(echo "#!/bin/bash
   echo ECS_CLUSTER=$CLUSTER_NAME >> /etc/ecs/ecs.config && yum update -y
+
+  # Install Dynatrace OneAgent client
+  yum install -y wget
+  wget -O Dynatrace-OneAgent-Linux-1.201.129.sh \
+    \"https://nhd42358.live.dynatrace.com/api/v1/deployment/installer/agent/unix/default/latest?arch=x86&flavor=default\" \
+    --header=\"Authorization: Api-Token 2S4ZegTjTj-De5OuVXqMc\"
```

Comment:
@kbcarter is it ok to have the API password here? I'm concerned about it being in the repo.  Maybe it should be an ENV variable and pushed to SSM?

Reply:
@kbcarter @dylansalay new key should be added here https://github.com/yalelibrary/yul-dc-camerata/blob/master/lib/camerata/secrets.rb#L13

For Dylan - you can access it with cam env_get VAR_NAME in the docs
For Keith - you can use ${VAR_NAME} in your script

Reply:
@orangewolf I moved the Dynatrace token an environment variable